backend/gs/file.go

```diff
@@ -200,8 +198,8 @@ func (f *File) MoveToLocation(location vfs.Location) (vfs.File, error) {
 // MoveToFile puts the contents of File into the targetFile passed using File.CopyToFile.
```

Comment:
not sure the comments need to be directly referencing variable names in the first place, but you changed the variable/arg name, and it no longer matches the one referenced in this comment. This should probably just say "...into the file argument passed..." or something similar.